Mashael_Almosallam/week_04/day_02/MTA/MTA.rb

```diff
@@ -0,0 +1,44 @@
+$line_N = ["Times Square", "34th", "28th", "23rd", "Union Square", "8th"]
+$line_L = ["8th", "6th", "Union Square", "3rd", "1st"]
+$line_6 = ["Grand Central", "33rd", "28th", "23rd", "Union Square", "Astor Place"]
+$stops_Count = 0
+
+def get_line_array(line)
```

Comment:
@MashaelAlmosallam I believe this function is gonna return back `nil`, as `$_line_N, $_line_L, $_line_6` don't exist, try `$line_N, $line_L, $line_6`